frontend/src/textToSpeech/textToSpeech.js

```diff
@@ -0,0 +1,380 @@
+                                            <p>{this.getCurrentSentence()}</p>
+                                        </div>
+                                    )
+                                    : null
+                            }
+                        </div>
+                        <div className='col-3'>
+                            <div className='box-missing-words'>
+                                <h2 className='para'>Missing Words:</h2>
+                                {
+                                    this.state.graded
+                                        ? (
+                                            <div className='row'>
+                                                <div className='col missing-words-list'>
+                                                    {missingWord.slice(0,
+                                                        Math.round(missingWord.length / 2))}
+                                                </div>
+                                                <div className='col missing-words-list'>{
+                                                    missingWord.slice(Math.round(missingWord.length
+                                                        / 2), missingWord.length)}
+                                                </div>
+                                            </div>
+                                        )
+                                        : null
+                                }
+                            </div>
+                        </div>
+                    </div>
+                    <div className='row'>
+                        <div className="col box">
+                            <form id='input-form'>
+                                <div className="form-group">
+                                    <label>
+                                    Please write exactly what you hear from the audio.
+                                    </label>
+                                    <textarea
+                                        className="form-control text-input"
+                                        id="content"
+                                        rows="10"
+                                        disabled={this.state.continue}>
+                                    </textarea>
+                                </div>
+                            </form>
+                        </div>
+                        <div className='col-2 sub-div' id="submit-btn-div">
+                            <button type="submit"
+                                className= 'btn btn-success submit-btn text-middle'
+                                form="picturebook-form"
+                                onClick={this.handleSubmit}
+                                disabled={this.state.continue}>
+                                Submit
+                            </button>
```

Comment:
I think this and the subsequent `<button>`s have the wrong `form` attribute. Looks like they might have a holdover from another component `picturebook-form` rather than pointing to the `input-form`.